config/install/apigee_edge.auth.yml

```diff
@@ -1 +1,2 @@
 active_key: ''
+oauth_token_storage_location: ''
```

Comment:
It should be set, you should not just configure an object in the scheme as string and you leave it as null by not defining that. Yes this could be the default location instead of an empty string, both options can be justified with good reasons why it should be like that. I would just leave it like this for now.